shared/utility/reactor/StreamReactor.hpp

```diff
@@ -443,6 +458,9 @@ namespace utility::reactor {
                     log<NUClear::WARN>(ex.what());
                 }
             }
+            if (log_level <= NUClear::TRACE) {
+                log<NUClear::TRACE>("Read:", debug_string.str());
+            }
```

Comment:
This one is here on purpose, should probably have a comment.
Since this will be dumping so much data to log you really don't want to be doing all the work involved in making a log line for every single read/write you get.